src/bb2.py

```diff
@@ -2,13 +2,90 @@
 Blue Button 2.0 SDK Class
 
 """
+import json
+import os
+import pathlib
+import yaml
+
+ROOT_DIR = os.path.abspath(os.curdir) + "/"
+DEFAULT_CONFIG_FILE_LOCATION = ROOT_DIR + "./.bluebutton-config.json"
+
+ENVIRONMENT_URLS = {
+    "SANDBOX": "https://sandbox.bluebutton.cms.gov",
+    "PRODUCTION": "https://api.bluebutton.cms.gov",
+}
 
 
 class Bb2:
     name = "bb2"
     verbose_name = "Blue Button 2.0 SDK Package"
-    
-    def hello(self):
-        mesg = "Hello from BB2 SDK Class method!!!"
-        print(mesg)
-        return mesg
\ No newline at end of file
+
+    def __init__(self, config_file=DEFAULT_CONFIG_FILE_LOCATION):
```

Comment:
So this constructor will only support a config file location, not passing in the config as a dict?

Reply:
@nbragdon I will add this in. That was something I liked in the node SDK that you added. TY!

Reply:
@nbragdon Just added the option to pass in a dictionary.